subsys/bluetooth/controller/hci/hci.c

```diff
@@ -21,6 +21,7 @@
 #include <zephyr/bluetooth/hci_vs.h>
 #include <zephyr/bluetooth/buf.h>
 #include <zephyr/bluetooth/bluetooth.h>
+#include <zephyr/bluetooth/conn.h>
```

Comment:
Why should the controller depend on conn.h? Isn’t that a purely host-specific API? What we discussed last week was that hci.h should get split into two files, so that one contains strictly only HCI protocol definitions (structs & defines) and nothing else.

Reply:
Will rework as per the Discord discussion.